sdk/core/test/unit/services/CognitoService.test.ts

```diff
@@ -711,16 +712,35 @@ describe('CognitoService', () => {
   })
 
   describe('#changeUsername', () => {
-    it(successPathTestName, async () => {
-      const error = { code: USER_NOT_FOUND_EXCEPTION }
+    describe('[username is not taken]', () => {
+      beforeEach(() => {
+        stubMethod(INITIATE_AUTH, null, { code: USER_NOT_FOUND_EXCEPTION })
```

Comment:
Shouldn't it be `beforeAll`?

Reply:
No, because it restores in `afterEach`:
https://github.com/affinityproject/affinidi-core-sdk/blob/65ed77a4e28177ae727fe638be9932073bb67f18/sdk/core/test/unit/services/CognitoService.test.ts#L54

Reply:
Oh. That's really hard to follow, because `afterEach` is on another level.
But since it was there before, let's leave it as it was.